src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -197,8 +216,9 @@ export const NavBar: React.FC = track(
               className="mobileHamburgerButton"
               borderLeft="1px solid"
               borderColor="black10"
-              px={1}
               ml={1}
+              tabIndex={0}
+              role="button"
```

Comment:
Really this thing shouldn't be using a `NavItem` but all the tracking is buried in it, so be it.

`role="button"` is what makes this visible to voiceover in absence of an `href`